site/docs/install-windows.md

```diff
@@ -5,69 +5,40 @@ title: Installing Bazel on Windows
 
 # <a name="windows"></a>Installing Bazel on Windows
 
-Supported Windows platforms:
+### Prerequisites
 
-*   64 bit Windows 7 or higher, or equivalent Windows Server versions.
+*   64 bit Windows 7 or newer, or <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832(v=vs.85).aspx">equivalent Windows Server versions</a>
 
-_Check
-<a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832(v=vs.85).aspx">Microsoft's
-Operating System Version table</a> to see if your OS is supported._
-
-### 1. Install prerequisites (if not already installed)
-
-*   Python 2.7 or later.
-
-    Use the Windows-native Python version. Do not use Python that comes with the
-    MSYS2 shell or that you installed in MSYS using Pacman because it doesn't
-    work with Bazel.
-
-*   [MSYS2 shell](https://msys2.github.io/).
-
-    You also need to set the `BAZEL_SH` environment variable to point to
-    `bash.exe`. For example in the Windows Command Prompt (`cmd.exe`):
-
-    ```
-    set BAZEL_SH=C:\tools\msys64\usr\bin\bash.exe
-    ```
-
-    **Note**: do not use quotes (") around the path like you would on Unixes.
-    Windows doesn't need them and it may confuse Bazel.
-
-*   Several MSYS2 packages.
-
-    Run the following command in the MSYS2 shell to install them:
-
-    ```bash
-    pacman -Syuu git curl zip unzip
-    ```
+*   [MSYS2 shell](https://msys2.github.io/)
 
 *   [Microsoft Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
 
-    This may already be installed on your system.
+    These are common DLLs that Bazel needs. You may already have them installed.
 
-### 2. Install Bazel on Windows using one of the following methods:
+### Getting Bazel
 
-*   [Download the binary (recommended)](#download-the-binary-recommended)
-*   [Install using Chocolatey](#install-using-chocolatey)
-*   [Compile Bazel from source](install-compile-source.html)
+Download Bazel for Windows from our
+[GitHub releases page](https://github.com/bazelbuild/bazel/releases).
+Look for `bazel-<version>-windows-x86_64.exe`, e.g. `bazel-0.13.0-windows-x86_64.exe`.
 
-#### Download the binary (recommended)
+**Tip:** For convenience, rename the downloaded binary to `bazel.exe` and move it to a directory
+that's on your `%PATH%` or add its directory to your `%PATH%`. This way you can run Bazel by
+typing `bazel` in any directory, without typing out the full path.
 
-Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases)
-and download the Windows binary<sup>1</sup>: `bazel-<version>-installer-windows-x86_64.sh`.
-
-For convenience, move the binary to a directory that's on your `%PATH%`. This
-way you can run Bazel by typing `bazel` in any directory, without typing out the
-full path. That said, you may put the binary anywhere on your filesystem.
+**Note:** Bazel includes an embedded JDK so you don't need to install one separately. If you want
```

Comment:
FYI, we'll no longer provide "without embedded JDK" binaries.
See https://github.com/bazelbuild/continuous-integration/issues/269

Reply:
Good to know, thanks! I'll remove this section once that PR is merged.